### ISSUE TITLE
Fraction of imputed values is zero in results.imputation.pdf

### ISSUE THREAD

**Issue by daniswan** [biodavidjm/artMS#104]
results.imputation.pdf output file from AnalysisQuantifications functions.  In image, the percentage/fraction of imputed values is shown as zero, when clearly it is not zero. 

![image](https://user-images.githubusercontent.com/20587010/57409518-587e7f00-719d-11e9-8307-0bd4e49115fc.png)

**Comment by biodavidjm** [comment]
Are you talking about the 0.00 / 1.00 on top of the chart? if so, that it is indicating the scale (from 0 to 1). That's why every quarter is 0.25, 0.50, 0.75... 1. Makes sense?

**Comment by daniswan** [comment]
Yes.  But from the look of it maybe 8% of data was imputes, so it should be
0.08/1.00 or something like that?  Or do I not understand?

**Comment by biodavidjm** [comment]
Well, just by using the reference axis, the proportion of imputed values should be a little bit less than 0.0625 (>6%). Can you check if that is the actual percentage of imputed values? I'll try to add the legend with the actual values.
